frontend/components/notifications/management/NotificationsManagementPairedDeviceModalContent.vue

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" setup>
+import { faTrash } from '@fortawesome/pro-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { type NotificationPairedDevice } from '~/types/api/notifications'
+
+// TODO: Implement handling of user input
+
+const { t: $t } = useI18n()
+
+interface Props {
+  device: NotificationPairedDevice
+}
+const props = defineProps<Props>()
+
+const notificationsToggle = ref(false)
+const accountAccessToggle = ref(false)
+
+</script>
+
+<template>
+  <div class="row-container">
+    <div class="device-row ">
+      <div class="device">
+        {{ $t('notifications.general.paired_devices.device') }}: {{ props.device.name || $t('notifications.general.paired_devices.unknown') }}
+      </div>
+      <Button data-secondary class="p-button-icon-only">
+        <FontAwesomeIcon :icon="faTrash" />
+      </Button>
```

Comment:
tell me what this button does
1. best way: `button-text` that is only perceivable via screen reader, see (sr-only): https://tailwindcss.com/docs/screen-readers
2. aria-label='Permanently remove 'My device' from the list.' 
3. title-attribute for svg

Reply:
Will be done in a separate ticket, just like mentioned here: https://github.com/gobitfly/beaconchain/pull/594#discussion_r1696747034